db_helper.py

```diff
@@ -10,9 +13,25 @@ def connectToDatabase():
     return connection
 
 
-# function that disconnects from the database
-def disconnectFromDatabase(connection):
-    connection.close()
+class ConnectionManager:
```

Comment:
I've tried changing the destination of this PR to try and get rid of this (since this now exists in AGENDUM-57) but I haven't had any luck...